bazel/repository_locations.bzl

```diff
@@ -913,13 +913,11 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Google Test",
         project_desc = "Google's C++ test framework",
         project_url = "https://github.com/google/googletest",
-        # Pick up fix for MOCK_METHOD compilation with clang-cl for Windows (resolved after 1.10.0)
-        # see https://github.com/google/googletest/issues/2490
-        version = "a4ab0abb93620ce26efad9de9296b73b16e88588",
-        sha256 = "7897bfaa5ad39a479177cfb5c3ce010184dbaee22a7c3727b212282871918751",
+        version = "7e33b6a1c497ced1e98fc60175aeb4678419281c",
```

Comment:
can we pin to a release version please?

Reply:
`1.14.0` is the latest - https://github.com/google/googletest/releases/tag/v1.14.0

Reply:
We can pin to a release version, but unfortunately not the latest (requires new absl and protobuf versions).